cypress/integration/pages/onDemandRadio/testsForCanonicalOnly.js

```diff
@@ -1,65 +1,70 @@
 import path from 'ramda/src/path';
+import config from '../../../support/config/services';
 import envConfig from '../../../support/config/envs';
 import { isScheduleDataComplete } from '../../../../src/app/containers/RadioSchedule/utilities/evaluateScheduleData';
 
 export default ({ service, pageType, variant }) => {
+  describe('Chartbeat', () => {
+    if (envConfig.chartbeatEnabled) {
+      it('should have a script with src value set to chartbeat source', () => {
+        cy.hasScriptWithChartbeatSrc();
+      });
+      it('should have chartbeat config set to window object', () => {
+        cy.hasGlobalChartbeatConfig();
+      });
+    }
+  });
   describe(`testsForCanonicalOnly for ${service} ${pageType} ${variant}`, () => {
     beforeEach(() => {
-      cy.getToggles(service);
+      cy.getToggles(config[service].name);
     });
 
-    describe('Chartbeat', () => {
-      if (envConfig.chartbeatEnabled) {
-        it('should have a script with src value set to chartbeat source', () => {
-          cy.hasScriptWithChartbeatSrc();
-        });
-        it('should have chartbeat config set to window object', () => {
-          cy.hasGlobalChartbeatConfig();
-        });
-      }
-    });
     describe('Radio Schedule', () => {
       it('should be displayed if there is enough schedule data', function test() {
-        const scheduleIsEnabled = path(
-          ['onDemandRadioSchedule', 'enabled'],
-          this.toggles,
-        );
-        cy.log(
-          `On Demand Radio Page configured for Radio Schedule? ${scheduleIsEnabled}`,
-        );
-
-        if (scheduleIsEnabled) {
-          const currentPath = Cypress.env('currentPath');
-
-          const masterBrand = currentPath.split('/')[2];
-
-          let schedulePath = `/${service}/${masterBrand}/schedule.json`.replace(
-            'bbc_afaanoromoo_radio',
-            'bbc_oromo_radio',
+        cy.fixture(`toggles/${config[service].name}.json`).then(toggles => {
+          const scheduleIsEnabled = path(
+            ['onDemandRadioSchedule', 'enabled'],
+            toggles,
+          );
+          cy.log(
+            `On Demand Radio Page configured for Radio Schedule? ${scheduleIsEnabled}`,
           );
-          if (Cypress.env('APP_ENV') === 'test') {
-            schedulePath += '?renderer_env=live';
-          }
 
-          cy.request(schedulePath).then(({ body: scheduleJson }) => {
-            const { schedules } = scheduleJson;
+          if (scheduleIsEnabled) {
```

Comment:
As with my other comment - can these schedule test changes be broken into their own PR?

Reply:
changed back